python/test/unit/language/test_core.py

```diff
@@ -7168,3 +7168,14 @@ def aliasing_kernel(buffer, buffer2):
     buffer = torch.zeros(1, device=device)
     aliasing_kernel[(1, )](buffer, buffer)
     assert buffer[0] == 1
+
+
+@pytest.mark.interpreter
+@pytest.mark.parametrize("dtype", map(tl.dtype, tl.dtype.SINT_TYPES + tl.dtype.UINT_TYPES + tl.dtype.STANDARD_FP_TYPES))
+def test_dtypes(device, dtype):
```

Comment:
You may want to differentiate this test with `test_dtype`. Either merge tests or change one of the function names.

Reply:
Ok, renamed it to `test_dtype_tensor` (willing to take suggestions on better names if you have any). I'm not entirely sure what `test_dtype` is supposed to do so I left it alone for now.